src/environmentd/tests/auth.rs

```diff
@@ -302,6 +302,7 @@ where
 // postgres-style error.
 enum Assert<E, D = ()> {
     Success,
+    SuccessSuperuserCheck(bool),
```

Comment:
I tried to update `Success` so that it could take queries and expected results as a parameter, but I ran out of patience trying to make it generic over `pgwire`, `HTTP`, and `ws`.